src/pattern-library/components/patterns/SharedMoleculePatterns.js

```diff
@@ -12,30 +12,35 @@ export default function SharedMoleculePatterns() {
     <PatternPage title="Molecules">
       <Pattern title="Frame">
         <p>
-          A <code>frame</code> has a border and a background, but no other
-          layout affordances.
+          A <code>frame</code> has a border, background, padding, and vertical
```

Comment:
A frame is looking more to a card

Reply:
A frame gives you a border, background, padding and spacing. A card gives you dimensionality (shadow/hover shadow). These patterns are still settling out a bit. The answer to 

> If I want to create a new component do I based on a frame or a card?

is currently "neither" — these patterns are still private to the pattern library. Once they're not private anymore this question should have a clear(er) answer.